### ISSUE TITLE
History hidden/deleted indicator idea

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#9997]
I find the big yellow warnings to be... big. and make scanning the history difficult.

I'd like to propose the following, e.g. 740. Place some markers to the left (or right side) of the dataset to indicate that it is hidden. The delete could be a drop down for undelete / delete permanently?

![image](https://user-images.githubusercontent.com/458683/87448693-7266ed80-c5fc-11ea-92c0-97fc39c8ea81.png)

**Comment by dannon** [comment]
Yeah, the big yellow warnings aren't great.  With #8441 happening very soon, it's a good opportunity to make sure this is updated.
